src/common/utils.c

```diff
@@ -108,6 +113,10 @@ static char* checkpath(char *path, const char *srcpath)
 {
 	// just make sure the char*path is not const
 	char *p = path;
+
+	nullpo_retr(NULL, path);
```

Comment:
According to the check below, `path` and `srcpath` seem like they can safely be NULL